src/de.rs

```diff
@@ -228,9 +230,24 @@ impl<'de, 'a> de::Deserializer<'de> for &'a mut Deserializer<'de> {
     fn deserialize_char<V>(self, visitor: V) -> Result<V::Value>
         where V: Visitor<'de>
     {
-        let parser = sym(b'\'') * take(1) - sym(b'\'');
+        let parser = sym(b'\'') * take(1);
```

Comment:
There is most likely a better way to do this with POM, but it didn't work for me and this is somehow easier to read. I wonder if there is a parser library that is fast and nice to read.